src/lib/Microsoft.Health.Fhir.R4.Ingest/Service/R4FhirHealthService.cs

```diff
@@ -0,0 +1,45 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using EnsureThat;
+using Hl7.Fhir.Rest;
+
+namespace Microsoft.Health.Fhir.Ingest.Service
+{
+    public class R4FhirHealthService :
+        FhirHealthService
+    {
+        private readonly IFhirClient _client;
+
+        public R4FhirHealthService(IFhirClient fhirClient)
+        {
+            _client = EnsureArg.IsNotNull(fhirClient, nameof(fhirClient));
+        }
+
+        public override Task<FhirHealthCheckStatus> CheckHealth()
+        {
+            try
+            {
+                Hl7.Fhir.Model.Bundle result = _client.Search<Hl7.Fhir.Model.StructureDefinition>();
```

Comment:
is there a SearchAsync we can use?  Alternately you can just use a /get since you aren't specifying any search parameters.  I would either switch to a get for efficiency or add a _count = 1 on the StructureDefinition search.

Reply:
Went with SearchAsync